sass/_variables.scss

```diff
@@ -1,10 +1,34 @@
 /* Colors */
-$paragraph-color: #000000;
+$paragraph-color: #000;
 $brand-main-color: #ff0000;
+$orange-button-color: #f58220;
+$section-bg-color: #FFF;
```

Comment:
jesli wszedzie kolory sa malymi literami to spojniej byloby zeby tutaj tez bylo malymi